internal/pkg/coordinator/monitor.go

```diff
@@ -188,10 +189,29 @@ func (m *monitorT) handlePolicies(ctx context.Context, hits []es.HitT) error {
 	return nil
 }
 
+func isAvailabilityError(e error) bool {
+	if e == nil {
+		return false
+	}
+	match, err := regexp.MatchString("connect: connection refused$|net/http: timeout awaiting response headers$", e.Error())
```

Comment:
This looks potentially error prone. Is there any other way to check for this like status code or similar?

Reply:
yeah, i agree, i suspect it'll be platform dependent, since i believe that these messages come from the OS level.

unfortunately these cases don't make a full http roundtrip, so there aren't proper status codes to check. we could do something internally. for example, we could check for a 0 status code, which is the default the status code will be set to coming out of libbeat.